eng/config/PublishData.json

```diff
@@ -134,6 +134,16 @@
       "vsMajorVersion": 17,
       "insertionCreateDraftPR": true,
       "insertionTitlePrefix": "[d17.4p3]"
+    },
+    "main-vs-deps": {
+      "nugetKind": [
+        "Shipping",
+        "NonShipping"
+      ],
+      "vsBranch": "main",
+      "vsMajorVersion": 17,
+      "insertionCreateDraftPR": true,
+      "insertionTitlePrefix": "[Validation]"
```

Comment:
This would be out d17.3p3 release branch. The main branch would be for Validation.

Reply:
Of course it is, why did I type that 😆